packages/frontend/editor-ui/src/features/projects/views/ProjectSettings.vue

```diff
@@ -528,50 +558,29 @@ onMounted(() => {
 					@validate="isValid = $event"
 				/>
 			</fieldset>
-			<fieldset v-if="isDirty" :class="$style.buttons">
```

Comment:
Non-blocker: Given that there is now no indicator for unsaved changes anymore, what do you think about adding a `beforeunload` listener, that will display a prompt to the user when they are about to leave the page without having saved their changes?